src/i18n.js

```diff
@@ -1,14 +1,23 @@
 import i18next from 'i18next';
-import { initReactI18next } from 'react-i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
 import HttpApi from 'i18next-http-backend';
+import { initReactI18next } from 'react-i18next';
+
+//fallback defined by ENV disabled because config cant be imported
+//import config from '/config/default';
+//const [defaultLanguage] = useState(config.get('settings.defaultLanguage'))
 
 i18next
     .use(initReactI18next)
     .use(HttpApi)
+    .use(LanguageDetector)
     .init({
-        // Remove resources from here
-        lng: 'en',
-        fallbackLng: 'en',
+        //Language with autodetection
+        fallbackLng: 'en', //{defaultLanguage}
```

Comment:
The thing is that I want the english language to be the default language for hemmelig.app, this means if the default language is not provided from the env vars, then it will use the browser language, and set the fallbackLng to "en" either way.

Reply:
So, if a default language is defined through env variable, this should disable browser language detection and **only** use this defined language? only if there is no language defined, language detection should be used? 

and then provide a language selection menu somewhere on the page.

Reply:
This has been added in the latest commit